app/src/pages/Courses.tsx

```diff
@@ -80,36 +124,122 @@ const Courses = () => {
           {data.length ? (
             <>
               {/* Header and create course button */}
-              <div className="flex flex-row no-wrap">
+              <div className="flex flex-row no-wrap items-center">
                 <h1 className="text-3xl font-bold flex-1">
                   Confira seus cursos
                 </h1>
                 <div className="flex flex-row gap-5">
                   <label
                     htmlFor="course-create"
                     onClick={CourseManager}
-                    className="std-button flex modal-button space-x-2 cursor-pointer"
+                    className="std-button flex modal-button space-x-2 cursor-pointer px-12 py-2.5"
                   >
-                    <PencilSquareIcon className="w-8 h-8" />
-                    <p className="font-normal ">Criar novo curso</p>{" "}
+                    <p className="font-normal flex items-center "><span className="text-3xl mr-2">+</span> Novo Curso</p>{" "}
```

Comment:
I feel like this could be split up into more lines for readability. A minor detail, though.